security/management/commands/set_celery_task_log_state.py

```diff
@@ -0,0 +1,27 @@
+from datetime import timedelta
+
+from django.core.management.base import BaseCommand
+from django.utils import timezone
+
+from security.enums import CeleryTaskRunLogState, CeleryTaskLogState
+from security.models import CeleryTaskLog
+
+
+from celery import current_app
+
+
+class Command(BaseCommand):
+
+    def handle(self, **options):
```

Comment:
this is major change. Old command was really slow, because filter_stale use high numberu CeleryTaskRunLog rows in exclude. The queryset was really really slow and ate whole DB free memotry

Reply:
99% SUCCEEDED or FAILED CeleryTaskLog instances will be set. Because of non atomic db connection. Only for applications that uses atomic connection for log will have to set state of these tasks log.